src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -389,188 +392,214 @@ public function getJobStatusValues(): array
         ];
     }
 
-    /**
-     * @return FieldList
-     */
-    public function getCMSFields()
+    public function getCMSFields(): FieldList
```

Comment:
It's recommended to view this diff with the hide whitespace setting enabled to better see the changes.

Reply:
Please revert this change. <ethod signatures must not change outside of a major release.